e2e/__tests__/override-globals.test.js

```diff
@@ -15,3 +15,14 @@ test('overriding native promise does not freeze Jest', () => {
   const run = runJest('override-globals');
   expect(run.stderr).toMatch(/PASS __tests__(\/|\\)index.js/);
 });
+
+test('has a duration even if time is faked', () => {
+  const regex = /works well \((\d+)ms\)/;
+  const {stderr} = runJest('override-globals', ['--verbose']);
+
+  expect(stderr).toMatch(regex);
+
+  const [, duration] = stderr.match(/works well \((\d+)ms\)/);
```

Comment:
Can you move the regex to a variable to avoid copy&pasting?

Reply:
Hah, thought I did that, see line 20... Will fix!